lighthouse-core/config/constants.js

```diff
@@ -38,6 +38,13 @@ const throttling = {
     uploadThroughputKbps: 700 * DEVTOOLS_THROUGHPUT_ADJUSTMENT_FACTOR,
     cpuSlowdownMultiplier: 4,
   },
+  // Using a "broadband" connection type
+  // Corresponds to "Dense 4G 25th percentile" in https://docs.google.com/document/d/1Ft1Bnq9-t4jK5egLSOc28IL4TvR-Tt0se_1faTA4KTY/edit#heading=h.bb7nfy2x9e5v
+  desktopDense4G: {
+    rttMs: 40,
```

Comment:
should we set `requestLatencyMs: 0, downloadThroughputKbps: 0, uploadThroughputKbps: 0` for no throttling when `--throttling-method=devtools`?

Reply:
would this be done in just the dt entry's `createConfig`?

Reply:
bump :)

Reply:
I assumed we'd set it on this same object like we do for mobile settings

Reply:
tbh I've lost the thread here and have no idea how to synthesize what your saying :)

Reply:
I'm saying this object should look like this

```
  desktopDense4G: {
    rttMs: 40,
    throughputKbps: 10 * 1024,
    requestLatencyMs: 0, // 0 means unset
    downloadThroughputKbps: 0,
    uploadThroughputKbps: 0,
    cpuSlowdownMultiplier: 1,
  }
```

assuming that 0 still means unset to devtools

Reply:
but you said `when --throttling-method=devtools`, so what about for when throttling is set to simulated? That's why I thought this might need to be done in createConfig, behind a check for the throttling method


and is this what we want for `lr-desktop-config` too?

Reply:
Well adding these here is just going to be a noop when `--throttling-method=simulate`, I was distinguishing that this object should cover both throttling cases like the mobile 4g ones do above.

Reply:
+1 to adding `=0`, imo is more explicit and matches the pattern of all of the ones above.

Reply:
did we decide not to do this? my suggestion seems to have a +1 from shane :)

Reply:
i forgot to push